charts/steward/values.yaml

```diff
@@ -48,30 +48,31 @@ pipelineRuns:
   logging:
     elasticsearch:
       indexURL: ""
-      splitMessagesLongerThan: 1048576
-    forwarding:
+      connectTimeoutMillis: ~
+      requestTimeoutMillis: ~
+      socketTimeoutMillis: ~
+      splitMessagesLongerThan: ~
+    forwarder:
       enabled: false
+      useSidecar: false
       host: ""
-      useSidecar: true
       port: '24224'
-      tag: BUILD_LOGS
-      ## mandatory parameters of the elasticsearch-logs-plugin
-      senderBaseRetryIntervalMillis: 1000
-      senderMaxRetryIntervalMillis: 1000
-      senderMaxRetryCount: 30
-      connectionTimeoutMillis: 3000
-      readTimeoutMillis: 3000
-      ## optional parameters of the elasticsearch-logs-plugin
-      #  flushAttemptIntervalMillis: 100
-      #  bufferChunkInitialSize: 1048576 # 1 MB
-      #  bufferChunkRetentionSize: 3145728 # 3 MB
-      #  maxBufferSize: 10485760 # 10 MB
-      #  bufferChunkRetentionTimeMillis: 1000
-      #  maxWaitSecondsUntilBufferFlushed: 30
-      #  maxWaitSecondsUntilFlusherTerminated: 30
-      #  emitMaxRetriesIfBufferFull: -1 # forever
+      tag: 'logs'
+      senderBaseRetryIntervalMillis: ~
+      senderMaxRetryIntervalMillis: ~
+      senderMaxRetryCount: ~
+      connectionTimeoutMillis: ~
+      readTimeoutMillis: ~
+      maxWaitSecondsUntilBufferFlushed: ~
+      maxWaitSecondsUntilFlusherTerminated: ~
+      bufferChunkInitialSize: ~
+      bufferChunkRetentionSize: ~
+      bufferChunkRetentionTimeMillis: ~
+      flushAttemptIntervalMillis: ~
+      maxBufferSize: ~
+      emitTimeoutMillis: ~
   jenkinsfileRunner:
-    image: "stewardci/stewardci-jenkinsfile-runner:230825_1c09345"
+    image: "stewardci/stewardci-jenkinsfile-runner:TEST_231002_8e67897"
```

Comment:
I'll comment here just as a reminder to update the tag once JFR is released.

Reply:
Set to `231006_5085985`.